test/edu-benefits/5490/e2e-test.js

```diff
@@ -155,11 +155,4 @@ const runTest = (client, onlyRequiredFields) => {
   client.end();
 };
 
-// Prevent the test from being run on production.
-// TODO: Remove when the form is actually launched.
-if (!process.env.BUILDTYPE || process.env.BUILDTYPE === 'development') {
-  module.exports = runTest;
-} else {
-  // Empty function to skip the tests
-  module.exports = (client) => client.end();
-}
+module.exports = runTest;
```

Comment:
Looks right, though just realized this file will go away with https://github.com/department-of-veterans-affairs/vets-website/pull/5342/files since we're eliminating the 'minimal' e2e tests.